File: Bookstore.Web/ClientApp/src/app/models/filter/printing-edition-filter.ts
import { CurrencyType } from "src/app/enums/currency-type";
import { PrintingEditionType } from "src/app/enums/edition-type";

export interface PrintingEditionFilter 
{
    types?: PrintingEditionType[]
    min?: number
    max?: number
    currency?: CurrencyType
    sort?: boolean
}

File: Bookstore.Shared/Models/Filter/PrintingEditionFilter.cs
using static Bookstore.Shared.Enums.Enums;

namespace Bookstore.Shared.Models.Filter
{
    public class PrintingEditionFilter
    {
        public PrintingEditionType[] Types { get; set; }

        public double? Min { get; set; }

        public double? Max { get; set; }

        public CurrencyType? Currency { get; set; }

        public bool? Sort { get; set; }
    }
}
